test/wallet-http-test.js

```diff
@@ -85,6 +85,30 @@ for (const witnessOpt of witnessOptions) {
       addr = Address.fromString(str, node.network);
     });
 
+    it('should enable seed phrase recovery', async () => {
+      const options = {
+        passphrase: 'PASSPHRASE',
+        mnemonic: 'zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo wrong'
+      };
+
+      const testwallet = await wclient.createWallet(`test_seed-${witnessOpt}`, options);
```

Comment:
Sorry one last nit: couldn't we just use `name` here each time instead of `test_seed-${witnessOpt}` ?

Reply:
I want to create a new wallet where I pass in the mnemonic at the time of creation. And I cannot create a wallet with the same name that has already been created.

Reply:
Ah right ok. Well you could define the wallet name at the top of this `it` block, since its not used in any other block:

```js
    const walletName = `test_seed-${witnessOpt}`;
```

I think this kind of thing just helps readability.